odbc_fdw.c

```diff
@@ -520,12 +584,69 @@ getQuoteChar(SQLHDBC dbc, StringInfoData *q_char)
     appendStringInfo(q_char, "%s", (char *) quote_char);
 }
 
+static void odbcConnStr(StringInfoData *conn_str,
+                        char *svr_dsn, char *svr_driver, char *svr_host, char *svr_port,
+                        char *svr_database, char *username, char *password)
+{
+  bool sep = FALSE;
+  static char *sep_str = ";";
+  initStringInfo(conn_str);
+  if (svr_dsn)
+  {
+    appendStringInfo(conn_str, "DSN=%s", svr_dsn);
+    sep = TRUE;
+  }
+  if (svr_driver)
+  {
+    if (sep)
+      appendStringInfoString(conn_str, sep_str);
```

Comment:
I agree with restyling the code. I'll set a separate issue to take care of that.